lib/content-services/src/lib/category/services/category.service.ts

```diff
@@ -58,14 +58,14 @@ export class CategoryService {
     }
 
     /**
-     * Creates subcategory under category with provided categoryId
+     * Creates subcategories under category with provided categoryId
      *
      * @param parentCategoryId The identifier of a parent category.
-     * @param payload Created category body
-     * @return Observable<CategoryEntry>
+     * @param payload List of categories to be created.
+     * @return Observable<CategoryPaging | CategoryEntry>
      */
-    createSubcategory(parentCategoryId: string, payload: CategoryBody): Observable<CategoryEntry> {
-        return from(this.categoriesApi.createSubcategory(parentCategoryId, [payload], {}));
+    createSubcategories(parentCategoryId: string, payload: CategoryBody[]): Observable<CategoryPaging | CategoryEntry> {
+        return from(this.categoriesApi.createSubcategory(parentCategoryId, payload, {}));
```

Comment:
As mentioned in previous comment that does not look correct because your function is "createSubcategories" so why createSubcategories calls createSubcategory which suggests creating single category and it returns even type for single result?

Reply:
`createSubcategory` in jsAPI receives a list of category bodies as second parameter, so maybe I should change name and return type in jsAPI for this function to `createSubcategories`? What do you think @AleksanderSklorz ?

Reply:
sounds like good idea